NSS-Wacky-Eels/BangazonSite
# Kr 4

## Link to Ticket
Closes #4 

## Description of Proposed Changes
- Created HomeTopTwenty product view
- Created method in productsController to return 20 products in descending order of creation date to the home page, while making the name of the product a hyper link to the products details

## Steps to Test
Check home page to see if top 20 items are there and the the name of the product when clicked takes you to the products details.

```
git fetch --all
git checkout <kr-4>

```


## Impacted Areas in Application
Home page

List general components of the application that this PR will affect:
*Products controller 
*HomeTopTwenty product view

## Mentions @username

@alexfont321
@priyankagargc27
@KaylaReid
@thatmikeparrish
@Akeen11

## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.
